hazelcast-client/src/test/java/com/hazelcast/client/cache/ClientCachingProviderTest.java

```diff
@@ -91,6 +92,17 @@ protected void assertInstanceStarted(String instanceName) {
         otherInstance.getLifecycleService().terminate();
     }
 
+    @Override
+    protected Collection<HazelcastInstance> getStartedInstances() {
+        return HazelcastClient.getAllHazelcastClients();
+    }
+
+    @Override
+    protected void cleanupForDefaultCacheManagerTest() {
+        clearSystemProperties();
+        clearCachingProviderRegistry();
```

Comment:
For the client-side cleanup before the tests for default `CacheManager`, just cleanup the `CachingProvider` registry and system properties; do not shutdown the `HazelcastInstance` member started in `@Before`.